.changeset/hot-lions-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-svelte': patch
+---
+
+Add notes about includes/excludes and path aliases
```

Comment:
personally i'd remove this changeset as we don't typically add them for documentation changes

Reply:
It's not documentation for the site, though, is it? I think this adds a comment to what people actually get when they run `create-svelte`, which would warrant a changeset.

Reply:
ah, yeah. fair enough